src/cosmospy/_transaction.py

```diff
@@ -2,13 +2,15 @@
 
 import base64
 import hashlib
-import json
-from typing import Any
 
 import ecdsa
 
 from cosmospy._wallet import DEFAULT_BECH32_HRP, privkey_to_address, privkey_to_pubkey
-from cosmospy.typing import SyncMode
+import cosmospy._interfaces.any_pb2 as Any
```

Comment:
```suggestion
from google.protobuf.any_pb2 import Any
```
Is this any different? With this we could delete one .proto file as well of course!

Reply:
Yes, it is. They lightly modified it. I think cosmospy would also work with the google any proto but I thought that if cosmos provides a custom one it would make sense to take it